src/plugins/views/editors/Flow/Components/Nodes/BaseNode/TreeView/TreeNode.js

```diff
@@ -397,9 +399,25 @@ class TreeNode extends BaseNode {
    *
    * @param {string} templateName node's template name
    */
-  onTemplateUpdate = templateName => {
+  onTemplateUpdate = template => {
+    const templateName = template.Label ?? template;
     if (templateName !== this.templateName) return; //not my template
     this._template = undefined;
+
+    if (this.isContainer) {
+      // Remove old belong line
+      this.removeBelongLine();
+      // Remove all children
```

Comment:
Why not having separate named functions to deal with these operations, instead of comments to label them? 
```suggestion
      this.removeChildren()
      this.removeCollapsibleItem();
      this.canvas.mInterface.graph.updateSubFlow(this, template);
```

Reply:
Thank you. Will do.